domains/alienriquebm.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "Portfolio's Ali Briceño",
+  "owner": {
+    "username": "alienriquebm",
+    "email": "alienriquebm@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://alienriquebm.github.io"
```

CNAME cannot have https or /

```suggestion
    "CNAME": "alienriquebm.github.io"
```